src/Microsoft.Identity.Web.TokenAcquisition/MergedOptions.cs

```diff
@@ -213,6 +213,9 @@ internal static void UpdateMergedOptionsFromMicrosoftIdentityOptions(MicrosoftId
             mergedOptions.StringDataFormat ??= microsoftIdentityOptions.StringDataFormat;
 #if NET8_0_OR_GREATER
             mergedOptions.TimeProvider = microsoftIdentityOptions.TimeProvider;
+#endif
+#if NET9_0_OR_GREATER
+            mergedOptions.AdditionalAuthorizationParameters = microsoftIdentityOptions.AdditionalAuthorizationParameters;
```

Comment:
>mergedOptions [](http://example.com/codeflow?start=12&length=13)

I am not familiar with this code, could you add to the PR description what is the impact of adding this? what is the use of merged options.

Reply:
This is for my education only :), thanks in advance

Reply:
I would recommend asking CoPilot integration in GitHub for these kinds of questions, for example:
![image](https://github.com/AzureAD/microsoft-identity-web/assets/19942418/81fdd58e-8abb-4f69-bbd2-9ff1701cb247)